library/init/meta/interactive.lean

```diff
@@ -665,7 +684,9 @@ private meta def find_case (goals : list expr) (ty : name) (idx : nat) (num_indi
   end else none
 
 private meta def rename_lams : expr → list name → tactic unit
```

Comment:
I feel like `rename_lams` should be reimplemented using `rename_many` and `rename_unstable` should be removed.

Reply:
Yes! That'll be the case tag PR which I'm working on, and which this PR is leading up to.

Reply:
:+1: